package dev.personal.planner.exceptions;

public class TaskNotFoundException {
}
